packages/codemirror/src/Editor.ts

```diff
@@ -32,6 +38,22 @@ export class Editor extends HTMLWidget {
         return this;
     }
 
+    markText(start: MarkTextPosition, end: MarkTextPosition): this {
+        if (this._codemirror) {
+            start.line--;
+            end.line--;
```

Comment:
Don't change the callers data.

Reply:
It would be unnatural to pass in `{line: 1, ch: 4}, {line: 1, ch: 5}` resulting in line 2 character 4 being highlighted. This is why I subtract one from the line input.